lametro/bills.py

```diff
@@ -50,9 +48,13 @@ def __init__(self, *args, **kwargs):
         self.scrape_restricted = True
 
     def _is_restricted(self, matter):
+        is_board_correspondence = matter['MatterTypeName'] in ('Board Box', 'Board Correspondence')
```

Comment:
checking for membership is classicaly the job of sets, so i think it's touch clearer like:

```python
matter['MatterTypeName'] in {'Board Box', 'Board Correspondence'}
```

Reply:
Thanks, @fgregg, done!